README.md

```diff
@@ -18,7 +18,7 @@ Best way to quickly integrate <a href="https://getbootstrap.com/">Bootstrap 3</a
 
 ## Links
 
-- [Documentation](http://valor-software.com/ngx-bootstrap/)
+- [Documentation](https://valor-software.com/ngx-bootstrap/)
```

Comment:
and this too?

Reply:
Up to you and the group. There's a push for web privacy so I thought it would be a good update to throw in.